notebooks/machine_learning/track_meta.py

```diff
@@ -0,0 +1,92 @@
+# See also examples/example_track/example_meta.py for a longer, commented example
+track = dict(
+    author_username='dansbecker',
+    course_name='Machine Learning',
+    course_url='https://www.kaggle.com/learn/machine-learning'
+)
+
+lessons = [ {'topic': topic_name} for topic_name in
+                    ['How Models Work',
```

Comment:
I'd suggest lower-casing these, because of the way they're dropped into sentences in `YOURTURN` and `KEEP_GOING`